package.json

```diff
@@ -73,6 +73,7 @@
     "http-proxy-agent": "2.1.0",
     "https-proxy-agent": "2.2.1",
     "jsonc-parser": "1.0.0",
+    "microsoft.aspnetcore.razor.vscode": "https://razorvscodetest.blob.core.windows.net/npm/microsoft.aspnetcore.razor.vscode-1.0.0-alpha1-20180924.3.tgz",
```

Comment:
We're consuming this via URL, rather than from npmjs.org, because there's no reason why any other code should want to take a dependency on it. There's just no reason for this to package to be on npmjs.org.